hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/domain/EntityIdServiceImpl.java

```diff
@@ -157,9 +149,15 @@ private EntityId load(AccountID accountId) {
                         findByEvmAddress(alias, accountId.getShardNum(), accountId.getRealmNum(), ACCOUNT) :
                         entityRepository.findByAlias(alias)
                                 .map(id -> EntityId.of(id, ACCOUNT))
-                                .orElseThrow(() -> new AliasNotFoundException(Hex.encodeHexString(alias), ACCOUNT));
+                                .orElseGet(() -> {
+                                    log.error(RECOVERABLE_ERROR + "Unable to find entity for alias {}",
+                                            Hex.encodeHexString(alias));
+                                    return null;
```

Comment:
```suggestion
                                    return EntityId.EMPTY;
```

looks like there's test case coverage issue as well

Reply:
I have updated it to `EntityId.EMPTY`.